Add support for not showing glfags help

denniszollo opened the pull request:
Refresh #206 for consideration.

Commit:
add support for not showing internal gflags help

Commit:
remove wrong define

denniszollo committed:
Remove extraneous line breaks in help string

RReichert committed:
SV4-144 fix google flags default notation

RReichert committed:
SV4-144 fix google flags default notation

schuhschuh commented:
Thanks for reviving (though I appreciate it's been over 1 year ago...).

Would you agree that alternative to a compile-time option, you would be interested in the equivalent of [declare_key_flag](https://github.com/abseil/abseil-py/blob/f0679ed8e79d1352f23b80965981b704bd48e1a4/absl/flags/_defines.py#L183)? And a change of `--help` to only display those key flags, and `--helpfull` to show both key and non-key flags? At the moment `--helpfull` and `--help` are equivalent for the gflags library.

jungleraptor committed:
fix: remove debug build from --version output (#2)

Fixes `Debug build` being printed when using --version.

We always compile with asserts enabled (-UNDEBUG), so checking
whether -NDEBUG is defined is not a reliable way of detecting
whether it is a release or debug build for us.

wwieclaw committed:
Revert "fix: remove debug build from --version output (#2)"

This reverts commit 40e90a1a4f48c386fde1ddc88546c6b3f04a01e6.

wwieclaw committed:
change how debug build is determined

wwieclaw committed:
fix

wwieclaw committed:
fix

wwieclaw committed:
update config name

wwieclaw committed:
fix

wwieclaw committed:
Merge pull request #3 from swift-nav/wwieclaw

restore "Debug Build" in  --version output

sys-ptipu commented:
**:closed_book: MEV SW OD Build <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/run_od_builds/27839/display/redirect>#27839</a> result: FAILURE**
 | Manifest Name | Build Status | Jenkins Link | Artifacts | Unsuccessful Stages | Description |
 | - | - | - | - | - | - |
 | mev-ts-trunk  | FAILURE  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-ts-trunk/59604/display/redirect>59604</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-ts-trunk/od/mev-ts-trunk-od-59604> Artifacts </a> | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-ts-trunk/59604/display/redirect>IPU SDK Build</a> |  |
 | mev-trunk  | FAILURE  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56680/display/redirect>56680</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-trunk/od/mev-trunk-od-56680> Artifacts </a> | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56680/display/redirect>Simics_C0_ACC</a> |  |
 | mev-release | NOT AFFECTED | | | |
 | mev-ts-release | NOT AFFECTED | | | |
 | mev-ts-release-1.6.1 | NOT AFFECTED | | | |
 | mev-ts-release-bitbake | NOT AFFECTED | | | |
 | mev-ts-release-external | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake-c1 | NOT AFFECTED | | | |
 | mev-ts-trunk-external | NOT AFFECTED | | | |
 | mev-upstream-idpf | NOT AFFECTED | | | |

sys-ptipu commented:
**:closed_book: MEV SW OD Build <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/run_od_builds/27894/display/redirect>#27894</a> result: FAILURE**
 | Manifest Name | Build Status | Jenkins Link | Artifacts | Unsuccessful Stages | Description |
 | - | - | - | - | - | - |
 | mev-trunk  | FAILURE  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56702/display/redirect>56702</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-trunk/od/mev-trunk-od-56702> Artifacts </a> | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-trunk/56702/display/redirect>IPU_IMC_6_6_4</a> |  |
 | mev-ts-trunk  | SUCCESS  | <a href=https://cje-ir-prod01.devtools.intel.com/nex-ncng-ipu/job/MEV/job/builds/job/official/job/mev-ts-trunk/59633/display/redirect>59633</a>  | <a href=https://ubit-artifactory-or.intel.com/artifactory/list/mountevans_sw_bsp-or-local/builds/official/mev-ts-trunk/od/mev-ts-trunk-od-59633> Artifacts </a> |  |  |
 | mev-release | NOT AFFECTED | | | |
 | mev-ts-release | NOT AFFECTED | | | |
 | mev-ts-release-1.6.1 | NOT AFFECTED | | | |
 | mev-ts-release-bitbake | NOT AFFECTED | | | |
 | mev-ts-release-external | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake | NOT AFFECTED | | | |
 | mev-ts-trunk-bitbake-c1 | NOT AFFECTED | | | |
 | mev-ts-trunk-external | NOT AFFECTED | | | |
 | mev-upstream-idpf | NOT AFFECTED | | | |

sys-ptipu commented:
mev-ts-trunk BAT tests: [success](http://10.166.204.21/test_session?id=59188) - build 59633 - it is required if PR have BAT MEV status check as mandatory. If failed, please review Tasks in provided link to Berta. For questions please contact with ipu.ci.triage.team@intel.com

adharsh committed:
Update CMake version to newer version

adharsh committed:
Merge pull request #5 from swift-nav/ababu/update_min_cmake_version

Update CMake minimum required version to 3.13

martin4861 committed:
Implement MODULE

martin4861 committed:
Merge pull request #6 from swift-nav/martin4861/implement-MODULE

Implement MODULE [AP-4070]

martin4861 committed:
Add dependency to rules_cc

martin4861 committed:
Merge pull request #7 from swift-nav/martin4861/add-dep-to-rules_cc

Add dependency to rules_cc

martin4861 committed:
QNX fix

martin4861 committed:
Merge pull request #8 from swift-nav/martin4861/qnx

QNX fix
